emoji/emoji_test.go

```diff
@@ -0,0 +1,32 @@
+package emoji
+
+import "testing"
+
+func Test_getEmojiUnicode(t *testing.T) {
```

Comment:
No need to _ just name it `TestGetEmojiUnicode `

Reply:
Done!